### ISSUE TITLE
Membership not deleted from users table in the database after group is removed

### ISSUE THREAD

**Issue by jeduardo** [getredash/redash#2055]
### Issue Summary

When a group is removed from Redash its id remains in the user membership list in the database.

A summary of the issue and the browser/OS environment in which it occurs.

### Steps to Reproduce

0. Before starting with the process, the database looks like this for 2 users and the default groups:

```
postgres=# select id, groups from users;
 id | groups 
----+--------
  2 | {2}
  1 | {1,2}
(2 rows)

postgres=# select id, name from groups;
 id |  name   
----+---------
  1 | admin
  2 | default
(2 rows)
```
1. Create a new group through the Redash web interface. It shows up on the database as expected. Membership is not yet assigned or changed.

```
postgres=# select id, groups from users;
 id | groups 
----+--------
  2 | {2}
  1 | {1,2}
(2 rows)

postgres=# select id, name from groups;
 id |      name       
----+-----------------
  1 | admin
  2 | default
 15 | Test Membership
(3 rows)
```
2. Assign one of the users to the new group through the Redash web interface. Membership is reflected in the database:

``` 
postgres=# select id, groups from users;
 id |  groups  
----+----------
  2 | {2}
  1 | {1,2,15}
(2 rows)

```

3. Delete the group using the Redash interface. The membership from the assigned user to the now non-existing group is not reflected in the database.

```
postgres=# select id, name from groups;
 id |  name   
----+---------
  1 | admin
  2 | default
(2 rows)

postgres=# select id, groups from users;
 id |  groups  
----+----------
  2 | {2}
  1 | {1,2,15}
(2 rows)
```
The user with id=1 remains a member of group 15 (Test Membership), however this group does not exist anymore.

I would expect that when the group is removed, membership references to the group are also removed.

### Technical details:

* Redash Version: 2.0.0+b2990
* Browser/OS: Chrome/Linux
* How did you install Redash: Docker compose (dev environment)

**Comment by kyoshidajp** [comment]
@arikfr Could you close this PR? I already fixed #2061. However, I didn't describe closing keyword like `Fixed`. Sorry.

**Comment by arikfr** [comment]
Fixed in #2061. Thank, @kyoshidajp .
